docs/maintainers/Release_Process.md

```diff
@@ -1,48 +1,64 @@
 < [Documentation](../Readme.md) / Release Process
 
-## Branches
+# Branches
 Our branching strategy is very similar to GitFlow
-* `master` branch has release commits. One commit - one release. Each release commit should be tagged with version `maj.min.patch`
-* `beta` branch is for stabilization of ongoing release. We don't merge new features into `beta` branch - only bug fixes are acceptable. Hotfixes for already released software should be delivered to this branch as well.
-* `develop` branch is a main branch for ongoing development. Pull requests with new features should be targeted to this branch, `develop` version is one step ahead of 'beta`
+- `master` branch has release commits. One commit - one release. Each release commit should be tagged with version `maj.min.patch`
+- `beta` branch is for stabilization of ongoing release. Only safe changes should be targeted into this branch. Breaking changes (e.g. save format changes) are forbidden in beta.
+- `develop` branch is a main branch for ongoing development. Pull requests with new features should be targeted to this branch, `develop` version is one major release ahead of `beta`
```

Comment:
Maybe I misunderstood this, does major mean increase major part of version?
We have "major.minor.patch"
Or it's different?

Reply:
Correct. However we never call 1.2.0 -> 1.3.0 updates as "minor".
We have "major releases", e.g. 1.3.0 and "hotfixes", e.g. 1.3.1.

If can think of better phrasing - sure, but neither "step" nor "minor release" are clear.
Perhaps something like "on develop, version is set to 1.)X+1).0 compared to beta"?

Reply:
True, as since bumping actual major version could look confusing (1->2) it maybe make sense to write down our versioning approach in the top, like
1(constant).major.patch unless we/community decide to change it

Reply:
Makes sense, e.g:
> VCMI releases use 1.X.Y format for versioning, where:
> X - major version
> Y - hotfix version

(but a bit more detailed)